CHANGELOG.md

```diff
@@ -1,5 +1,66 @@
 # Logic Apps Designer
```

Comment:
We're not calling the designer "Logic Apps Designer" anymore because the latest designer isn't labeled and "Logic Apps Designer" is the label on the designer for Consumption logic apps. We can just use "designer" or "workflow designer'.

```suggestion
# Azure Logic Apps - Workflow designer
```